docs/Tutorials/Syntax/Quest-Packages.md

```diff
@@ -0,0 +1,360 @@
+
+<div class="grid" markdown>
+!!! example "Example 1"
+    * :material-folder-open: QuestPackages
+        - :material-folder-open: woodCuttingQuest
+            - :material-file-star: package.yml
+            - {==:material-folder-open: miningQuest==}
+                - {==:material-file-star: package.yml==}
+
+!!! example "Example 2"
+    * :material-folder-open: QuestPackages
+        - :material-folder-open: woodCuttingQuest
+            - :material-file-star: package.yml
+              - {==:material-folder-open: miningQuest==}
+                - {==:material-file-star: package.yml==}
+                  - {==:material-folder-open: fishingQuest==}
+                    - {==:material-file-star: package.yml==}
+</div>
+
+Not hard to understand but to make things clearer:
+Every folder that contains a _package.yml_ file is a standalone quest package.
+By creating a folder with a _package.yml_ file inside another package, you create a standalone "subpackage". 
+
+--8<-- "Tutorials/download-complete-files.md"
+    ```
+    /bq download BetonQuest/Quest-Tutorials main QuestPackages /Syntax/packageStructure/3-PackageInPackage /packageStructure/PackageInPackage
+    ```
+    You can now find all files needed for this tutorial in this location:
+    "_YOUR-SERVER-LOCATION/plugins/BetonQuest/QuestPackages/packageStructure/PackageInPackage_"
+
+## 5. Creating subdirectories in a package
+
+Now you know, that it is possible to do nearly everything to structure your quests,
+But there is one more possibility: _subdirectories_.
+Let me first show you the example for it:
+
+!!! success "Example"
+    * :material-folder-open: QuestPackages
+        - :material-folder-open: woodCuttingQuest
+            - :material-file: package.yml
+            - {==:material-folder-open: questPart1==}
+                - :material-file: myEventsPart1.yml
+                - :material-file: MageConversation.yml
+            - {==:material-folder-open: questPart2==}
+                - :material-file: myEventsPart2.yml
+                - :material-file: KingConversation.yml
+
+You can create as many subdirectories in your quest packages as you like.
+Just create a folder with a name of your choice and start adding files to it!
+
+--8<-- "Tutorials/download-complete-files.md"
+    ```
+    /bq download BetonQuest/Quest-Tutorials main QuestPackages /Syntax/packageStructure/4-SubDirectory /packageStructure/SubDirectory
+    ```
+    You can now find all files needed for this tutorial in this location:
+    "_YOUR-SERVER-LOCATION/plugins/BetonQuest/QuestPackages/packageStructure/SubDirectory_"
+
+
+## Summary
+
+In this tutorial about quest packages tutorial you've learned how to efficiently organise your quests.
+ Let's sum it up: 
+
+* Quest packages are defined by a _package.yml_ file.
+* You can create a quest package by creating a folder and adding a _package.yml_ file to it.
+* Files inside quest packages (apart from the _package.yml_) do not follow any naming conventions.
```

Comment:
Files and Folders?

Reply:
But for folders with a package.yml file there is a limitation. No Spaces. A space will make it like it is a nested folder means
`My Package` is like `My-Package` as a address, while `MyPackage` is still `MyPackage`